src/components/pages/access_groups/AccessGroup.js

```diff
@@ -148,16 +148,42 @@ class AccessGroup extends React.Component {
 
   Actions() {
     return (
-      <div className="actions-wrapper">
-        <div className="actions-container content-lookup-actions-container">
-          <Action type="link" to={Path.dirname(this.props.match.url)} className="secondary" >Back</Action>
-          <Action type="link" to={UrlJoin(this.props.match.url, "edit")} hidden={!this.props.groupStore.accessGroup.isOwner}>Manage</Action>
-          <Action type="link" to={UrlJoin(this.props.match.url, "add-member")} hidden={!this.props.groupStore.accessGroup.isManager}>Add Member</Action>
-          <Action className="danger" onClick={this.LeaveAccessGroup} hidden={this.props.groupStore.accessGroup.isOwner}>Leave Group</Action>
-          <Action className="danger" onClick={this.DeleteAccessGroup} hidden={true || !this.props.groupStore.accessGroup.isOwner}>Delete</Action>
-          <ContentLookup />
-        </div>
-      </div>
+      <ActionsToolbar
+        actions={[
+          {
+            label: "Back",
+            type: "link",
+            path: Path.dirname(this.props.match.url),
+            className: "secondary"
+          },
+          {
+            label: "Manage",
+            type: "link",
+            hidden: !this.props.groupStore.accessGroup.isOwner,
+            path: UrlJoin(this.props.match.url, "edit"),
+          },
+          {
+            label: "Add Member",
+            type: "link",
+            hidden: !this.props.groupStore.accessGroup.isManager,
+            path: UrlJoin(this.props.match.url, "add-member")
+          },
+          {
+            label: "Leave Group",
+            type: "button",
+            hidden: this.props.groupStore.accessGroup.isOwner,
+            onClick: this.LeaveAccessGroup,
+            className: "danger"
+          },
+          {
+            label: "Delete",
+            type: "button",
+            hidden: (true || !this.props.groupStore.accessGroup.isOwner),
```

Comment:
@elv-kevin Do we need to keep this?

Reply:
Deleting an access group? May as well keep it there because it ought to be there, but it should not be visible because deleting a group doesn't work. Maybe someday it will.